src/main/kotlin/assimp/format/md3/Md3Loader.kt

```diff
@@ -752,8 +753,9 @@ class Md3Importer : BaseImporter() {
 
         // If no specific dir or file is given, use our default search behaviour
         if (configShaderFile.isEmpty()) {
-            if (!Q3Shader.loadShader(fill, "$path../../../scripts/$modelFile.shader"))
-                Q3Shader.loadShader(fill, "$path../../../scripts/$filename.shader")
+            // TODO read from memory: how do we resolve ../../..
+            if (!Q3Shader.loadShader(fill, "$path../../../scripts/$modelFile.shader", ioSystem))
+                Q3Shader.loadShader(fill, "$path../../../scripts/$filename.shader", ioSystem)
```

Comment:
If we don't find a solution now, we can postpone the decision until someone needs this and find together a solution

Reply:
Well I see 3 solutions:
1. leave it as it is, there is a simple workaround I used in the test
2. modify the ioSystem so that it resolves ".." in a way that it removes the last directory
3. change the ioSystem so it uses `java.nio.file.Path` instead of `Strings` to represent a file. That way resolving paths would be trivial, but implementing `Path` and `FileSystem`/`FileSystemProvider` takes a lot of effort and I don't think it's worth it.

I think for now we can leave it and maybe implement option 2 later.